### PR TITLE
[PLAY-1204] File Upload Kit Example Within Form

### PR DESCRIPTION
**Pull request by ElisaShapiro**
**What does this PR do?** A clear and concise description with your runway ticket url.
[PLAY-1204](https://nitro.powerhrg.com/runway/backlog_items/PLAY-1204) involves the creation of a **NEW** functional file upload kit form example (first prototyped in playgrounds [here](https://github.com/powerhome/playgrounds/tree/PLAY-1204)) with a description that explicitly states how to use the global prop `input_options` to match the `id` to the exist input as the current description of the custom example hints at. Also added console logging of uploaded file prior to submission per Jasper's commented suggestion. Also added `full_width: true` kit prop to all three File Upload examples as per Jason's suggestion on Runway Ticket.

**Screenshots:** Screenshots to visualize your addition/change
Rails file upload custom section

**How to test?** Steps to confirm the desired behavior:
1. Go to [File Upload kit page rails form example](https://playbook.powerapp.cloud/kits/file_upload/rails#form)
2. Try to upload a file:
     - Type file name in text input.
     - Upload file section should allow you to upload something from computer. 
     - Submission of upload (clicking Upload file primary button) should just reload the page.
3. Click on '</> Show Code' to see updated example code

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
~~- [ ] **TESTS** I have added test coverage to my code.~~

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_file_upload/docs/_file_upload_custom.html.erb [hunk 1]
```diff
@@ -1,7 +1,5 @@
-<%= pb_rails("file_upload", props: {id: "import_file"}) do %>
-  <input
-    id="import_file"
-    type="file"
-    name="import_file"
-  />
-<% end %>
\ No newline at end of file
+<%= pb_form_with(method: :get, multipart: true) do |form| %>
+  <%= form.text_field :name, style: "width: 140px", props: { placeholder: "Name" } %>
```
Comment (ElisaShapiro) [inline playbook/app/pb_kits/playbook/pb_file_upload/docs/_file_upload_custom.html.erb:2]: As discussed in 3/28 deep dive dailies review - name field is present as it is required for the playground demo to work and shows how this kit functions within a form/form group. But I can definitely remove that field from this faux-functional demo if simpler is better for a kit example (or just better aesthetically). Would look like:
Remove name field from example area

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_file_upload/docs/_file_upload_custom.html.erb:2]: Yeah I'd go ahead and remove the name field. @ElisaShapiro

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_file_upload/docs/_file_upload_custom.html.erb:2]: I'm fine logging the "File" that the user uploads.

Comment (ElisaShapiro) [reply playbook/app/pb_kits/playbook/pb_file_upload/docs/_file_upload_custom.html.erb]: Just pushed a commit that implemented this (as discussed in Connect). Thanks for the suggestion!

**Pull request closed without merging**